src/DataMapper/Product/PriceDataMapper.php

```diff
@@ -13,18 +14,15 @@
 use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
 use Sylius\Component\Core\Model\ChannelInterface;
 use Sylius\Component\Core\Model\ProductInterface;
-use Sylius\Component\Core\Model\ProductVariantInterface;
 use Sylius\Component\Currency\Converter\CurrencyConverterInterface;
 use Webmozart\Assert\Assert;
 
-/**
- * This data mapper maps prices on product documents
- */
```

Comment:
Is it so obvious? 😂

Reply:
I seems so, it's a **PriceDataMapper**, what else can it do? 😄